libbpf-tools/biosnoop.bpf.c

```diff
@@ -93,15 +96,21 @@ int trace_rq_start(struct request *rq, bool insert)
 }
 
 SEC("tp_btf/block_rq_insert")
-int BPF_PROG(block_rq_insert, struct request_queue *q, struct request *rq)
+int BPF_PROG(block_rq_insert)
 {
-	return trace_rq_start(rq, true);
+	if (LINUX_KERNEL_VERSION > KERNEL_VERSION(5, 10, 0))
```

Comment:
please leave comment explaining why we need to do this, similar to https://github.com/iovisor/bcc/pull/3317

same for bitesize.bpf.c below